src/python/grapl-template-generator/grapl_template_generator/grapl-templates/rust-grpc-service/{{cookiecutter.project_slug}}/tests/common/mod.rs

```diff
@@ -0,0 +1,70 @@
+use test_context::AsyncTestContext;
+use {{cookiecutter.snake_project_name}}::client::{Channel, Timeout};
+use tonic::transport::NamedService;
+use {{cookiecutter.snake_project_name}}::server::{{cookiecutter.service_name}}RpcServer;
+use {{cookiecutter.snake_project_name}}::server::{{cookiecutter.service_name}};
+use std::time::Duration;
+use tonic_health::proto::health_client::HealthClient;
+use tonic_health::proto::HealthCheckRequest;
+use tonic_health::proto::health_check_response::ServingStatus;
```

Comment:
- I agree with your assertion that an auto-format should occur at some point after project generation

- However, let's think outside the box a bit - I think it should be a precommit or prepush hook, which could potentially solve a whole class of "oh god damnit I forgot to lint/format" failures instead of this one narrow one

thoughts?

Reply:
If all our formatting executed quickly, that might be good. However, with all our Docker-based stuff, that's not always the case. The fact that such a hook would necessarily have to format _everything_ is even more frustrating (e.g., if I'm only working on Pulumi code, why should I have to wait for it to format a bunch of JavaScript and Rust I didn't touch?).

Any time such hooks have taken more than a second to run, I just end up disabling them because they're so frustrating.

Reply:
I'd prefer not to tackle this for now until. Ultimately, `make format` is not such an awful thing to subject developers to.